docfx.json

```diff
@@ -191,7 +191,8 @@
         "docs/framework/additional-apis/**/**.md": "managed-reference",
         "docs/framework/unmanaged-api/**/**.md": "reference",
         "docs/fsharp/language-reference/**/**.md": "language-reference",
-        "docs/standard/**/*how-to*.md": "how-to"
+        "docs/standard/**/*how-to*.md": "how-to",
+        "docs/whats-new/*.md" : "overview"
```

Comment:
I'm not sure this is the best choice. If you have suggestions @tdykstra let me know.

Reply:
my suggestion is "landing-page" to get the most appropriate KPI Score formula -- I've sent a query asking if we can use that value on a page that isn't strictly speaking a landing page.

Reply:
Kelly is still looking into creating a new type -- until that happens, what's new docs will have to remain as conceptual.

Reply:
It doesn't look like creating a new type is going to happen anytime soon -- should we close this PR?